client/src/components/page/Home.js

```diff
@@ -39,11 +35,13 @@ class Home extends Component {
         </p>
         <p>
           Below you will find a map of Googleplex in Mountain View, California.
+          You will also see some of the popular landmarks on the Google campus.
         </p>
-        <GoogleMapReact
-          bootstrapURLKeys={GOOGLE_MAPS_APP_KEY}
-          defaultCenter={GOOGLEPLEX_COORD}
-          defaultZoom={DEFAULT_MAP_ZOOM}
+        <HomeMap.HOME_MAP
```

Comment:
This map should probably be called `GoogleMaps` or `CustomMaps` instead of `HomeMap.HOME_MAP`. This map should take in a list of Markers. The markers should probably be defined in a folder probably something like `component/markers`.

The reason for this structure is that UI components should never correspond with only one page. We always try to make components as reuseable as possible otherwise the code will get messy with a lot of maps. If we take in a list of markers, then we can reuse this map for restaurants and food locations in the future on other pages.